modules/AOP_Case_Updates/AOP_Case_Updates.php

```diff
@@ -224,15 +224,28 @@ public function getEmailForUser()
 
         return array();
     }
+    /**
+     * @return array
+     */
+    public function getInformationForUser()
+    {
+        $user = $this->getUser();
+        if ($user) {
+            return array("lang" => $user->language_c, "emails" => array( $user->emailAddress->getPrimaryAddress($user)));
+        }
+
+        return array();
+    }
 
     /**
      * @param EmailTemplate $template
      * @param bool          $addDelimiter
      * @param null          $contactId
+     * @param string        $txtDelimiter
      *
      * @return array
      */
-    private function populateTemplate(EmailTemplate $template, $addDelimiter = true, $contactId = null)
+    private function populateTemplate(EmailTemplate $template, $addDelimiter = true, $contactId = null, $txtDelimiter = "" )
```

Comment:
Avoid changing a function signature whenever possible as this may break customizations.

In this case there is not need to change the signature, instead of 
```
$template->(args, $textDelimiter)
```
you can do
```
$template->txtDelimiter = foo
$template->(args)
```